standalone-metastore/metastore-server/src/main/resources/package.jdo

```diff
@@ -101,6 +101,18 @@
         <!-- allows null is true to keep backwards compatibility with old releases -->
         <column name="CREATE_TIME" jdbc-type="integer" allows-null="true"/>
       </field>
+      <field name="parameters" table="CATALOG_PARAMS">
+        <map key-type="java.lang.String" value-type="java.lang.String"/>
+         <join>
+           <column name="CTLG_ID"/>
+         </join>
+         <key>
+           <column name="PARAM_KEY" length="180" jdbc-type="VARCHAR"/>
+         </key>
+         <value>
+          <column name="PARAM_VALUE" length="32672" jdbc-type="VARCHAR"/>
```

Comment:
where does this value come from, is it a typo? DB PARAM_VALUE length is 4000 + in schema you also set 4000